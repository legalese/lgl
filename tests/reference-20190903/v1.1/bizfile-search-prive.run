LGL_URI=http://localhost/api/corpsec/v1.1 lgl bizfile search prive
