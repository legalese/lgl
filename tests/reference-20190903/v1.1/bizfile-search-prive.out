[ { '201711130R': 'PRIVE ADVISORY PTE. LTD.' },
  { '201539868D': 'PRIVE AESTHETICS (NEX) PTE. LTD.' },
  { '201321391R': 'PRIVE AESTHETICS PTE. LTD.' },
  { '201900970R': 'PRIVE ASIA PTE. LTD.' },
  { '201904025C': 'PRIVE ASSETS HOLDING PTE. LTD.' },
  { '201506783R': 'PRIVE CAPITAL GROUP PTE. LTD.' },
  { '201529821W': 'PRIVE CAPITAL MANAGEMENT PTE. LTD.' },
  { '201634710Z': 'KABINET PRIVE PTE. LTD.' } ]
